homeassistant/components/hassio/__init__.py

```diff
@@ -232,6 +244,21 @@ async def async_get_addon_discovery_info(
     return next((addon for addon in discovered_addons if addon["addon"] == slug), None)
 
 
+@bind_hass
+@api_data
+async def async_create_snapshot(
+    hass: HomeAssistantType, payload: dict, partial: bool = False
+) -> dict:
+    """Create a partial snapshot.
```

Comment:
```suggestion
    """Create a snapshot.
```
This function covers both full and partial